### DISCUSSION TITLE
Odd wide_storage

### DISCUSSION THREAD

**Discussion by DramaDesu** [jfalcou/eve#1892] [Q&A] [answered]
I'm wondering If I could declare eve::wide, but with odd num of wide storages.
I could write my own wrapper, but it seems to me like a bug.
```
	using wide_t = eve::wide<float>;
	using wide_triangles_t = eve::wide<float, eve::fixed<wide_t::size() * 3>>;

	wide_triangles_t triangles{};
```
It should create array of 3 regs, but this is incompatible because the static check is for powers of two for Cardinal.

**Comment by DramaDesu** [comment]
If it's a bug, I'll try to fix it.

**Reply by DramaDesu** [reply]
Motivation: I need this because every three elements depend on each other. 
And if I need to put data into registers one after another, I can do it with only three registers. 

It doesn’t matter what size the register is, you can always put a Cardinal of three-dimensional vectors in three registers and fill them completely.

**Comment by jfalcou** [comment, marked as answer]
You want to make a `eve::wide<kumi::tuple<float,float,float>>`
See the tutorial.about wide ofbstructs in the doc

**Reply by DramaDesu** [reply]
Hello, thank you, I'll try this

**Reply by DramaDesu** [reply]
https://jfalcou.github.io/eve/structeve_1_1struct__support.html
I don't understand how I missed it.
Many thanks!
